ntbs-service/Pages/ClinicalTimelines/Edit.cshtml.cs

```diff
@@ -57,8 +57,21 @@ public async Task<IActionResult> OnGetAsync(int? id)
             return Page();
         }
 
-        public async Task<IActionResult> OnPostAsync(int? id)
+        public async Task<IActionResult> OnPostPreviousPageAsync(int? NotificationId)
         {
+            await validateAndSave(NotificationId);
+
+            return RedirectToPage("/Patients/Edit", new {id = NotificationId});
+        }
+
+        public async Task<IActionResult> OnPostNextPageAsync(int? NotificationId)
+        {
+            await validateAndSave(NotificationId);
+
+            return RedirectToPage("/ClinicalTimelines/Edit", new {id = NotificationId});

```

Comment:
Shouldn't we redirect to a different page here to the current one? I know the next one doesn't exist yet, but just Patients/Index as before might make sense?